lib/python/treadmill_aws/bootstrap/zookeeper/aws.linux/treadmill/bin/run.sh

```diff
@@ -3,5 +3,4 @@
 SCRIPT_NAME="${0##*/}"
 SCRIPT_DIR="${0%/$SCRIPT_NAME}"
 
-{{ mount }} --make-rprivate /
-exec {{ pid1 }} -m -p "${SCRIPT_DIR}/run_real.sh"
+${SCRIPT_DIR}/run_real.sh
```

Comment:
the reason to have run_real.sh is because we needed to exec into something using pid1/chroot. you no longer need this, so i figure run_real.sh can be merged into run.sh - just one script?

Reply:
makes sense. merged it

Reply:
makes sense. merged it